### DISCUSSION TITLE
python3.92 torch 2.11 gradio 3.50 前端报错帮忙看一下是python的问题吗

### DISCUSSION THREAD

**Discussion by zbg2cd99** [zai-org/ChatGLM3#544] [General]
![image](https://github.com/THUDM/ChatGLM3/assets/40620710/0b77602a-a5a4-4603-87b2-a6dc744dc00c)
![image](https://github.com/THUDM/ChatGLM3/assets/40620710/c7ebada7-988b-439c-b7ee-40f9fa83b70a)

**Comment by zRzRzRzRzRzRzR** [comment]
降级到3.39 gradio
